demo-shell-ng2/src/app/components/activiti/activiti.component.ts

```diff
@@ -101,6 +109,7 @@ export class ActivitiComponent implements AfterViewInit, OnDestroy, OnInit {
     layoutType: string;
     currentTaskId: string;
     currentProcessInstanceId: string;
+    emptyListImageUrl: string = require('./../../../../../ng2-components/ng2-activiti-tasklist/src/assets/images/empty_doc_lib.svg');
```

Comment:
we don't need it

Reply:
Don't we need to show the image also with our custom template?